src/graph_compute/bsp_scheduling.h

```diff
@@ -1,7 +1,7 @@
 #ifndef BSP_SCHEDULING_H_
 #define BSP_SCHEDULING_H_
 
-#if D0_BSP
+#if D0_BSP || BASELINE
```

Comment:
BASELINE is not BSP, BASELINE is priority scheduling -- see `priority_scheduling.h`